src/main/java/core/basesyntax/service/impl/CsvReaderImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvReader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+public class CsvReaderImpl implements CsvReader {
```

Comment:
The class CsvReaderImpl should not begin with an empty line according to the common mistakes checklist.